chore: add info about destroying the network between config changes (#327)
vegaprotocol/vegacapsule: readme.md

vegacapsule network destroy --home-path=/var/tmp/veganetwork/testnetwork
```

> > ℹ️ NOTE: Capsule preserves some files when starting and stopping the network, for example any pre-generated keys, the genesis file, and any node configurations in the [network configuration file](https://github.com/vegaprotocol/vegacapsule/tree/main/net_confs). In order to start network with new values in these files, use  the `vegacapsule network destroy` command.

Author: ```suggestion
> ⚠️ Information:
> Capsule preserves some files when starting and stopping the network, for example any pre-generated keys, the genesis file, and any node configurations in the [network configuration file](https://github.com/vegaprotocol/vegacapsule/tree/main/net_confs). In order to start network with new values in these files, use  the `vegacapsule network destroy` command.
```

> > ℹ️ NOTE: Capsule preserves some files when starting and stopping the network for example any pre-generated keys, the genesis file or any node configurations in the the [network configuration file](https://github.com/vegaprotocol/vegacapsule/tree/main/net_confs) being used. In order to start network with new values in these files the `vegacapsule network destroy` command must be used.

Reviewer: ```suggestion
> ℹ️ NOTE: Capsule preserves some files when starting and stopping the network, for example any pre-generated keys, the genesis file, and any node configurations in the [network configuration file](https://github.com/vegaprotocol/vegacapsule/tree/main/net_confs). In order to start network with new values in these files, use  the `vegacapsule network destroy` command.
```

## Configuration

> Capsule can bootstraps network based on configuration. Please see `config.hcl` for example.

Reviewer: ```suggestion
Capsule can bootstrap a network based on configuration. Please see `config.hcl` for example.
```

Would it be correct to say "Capsule can bootstrap a network based on specific configurations that you can set"?

Reviewer: I realise this one is technically out of the scope of the PR but since I was here anyway... :)
